include/mesh.tcc

```diff
@@ -1455,7 +1451,7 @@ bool mpm::Mesh<Tdim>::read_particles_hdf5(unsigned phase,
   // Calculate the size and the offsets of our struct members in memory
   hsize_t nrecords = 0;
   hsize_t nfields = 0;
-  auto err = H5TBget_table_info(file_id, "table", &nfields, &nrecords);
+  H5TBget_table_info(file_id, "table", &nfields, &nrecords);
```

Comment:
some of these booleans would be useful to check if it is true, if not we return an error.

Reply:
Okay, I will revert and make a throw for some of them. Or do you want me to do for all of them?

Reply:
I give a second look on this. It returns negative integer if query is not successful. See: https://support.hdfgroup.org/HDF5/doc1.8/HL/RM_H5TB.html#H5TBget_table_info

However, in the next two lines, you are checking whether `(nfields != mpm::hdf5::particle::NFIELDS)`. So I don't think we need to use another if check.